react/MonthPicker/NativeMonthPicker/NativeMonthPicker.js

```diff
@@ -81,18 +86,22 @@ export default class NativeMonthPicker extends Component {
 
     return (
       <div className={rootClasses}>
+        <FieldLabel
+          {...{ id: `${id}`, label: label ? <span>{label} <ScreenReaderOnly>Month Year</ScreenReaderOnly></span> : <ScreenReaderOnly>Month Year</ScreenReaderOnly>, labelProps, secondaryLabel, tertiaryLabel }}
+        />
         <ChevronIcon
           className={styles.chevron}
           svgClassName={styles.chevronSvg}
           direction="down"
         />
         <input
-          {...(id ? { id } : {})}
+          id={id}
```

Comment:
Is `id` now a required property?

Reply:
Arrrr it would be nice but not sure I want to make that change - will revert.